vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -699,6 +699,17 @@ def list_models(cls):
         """
         return tuple(_SUPPORTED_MODELS)
 
+    @staticmethod
+    def _get_num_gpus_available():
+        # FIXME: update to use only torch for TIMM or find a better GPU detection strategy
+        # FIXME: get_gpu_count by itself doesn't always work for Windows
+        num_gpus = get_gpu_count()
+        if num_gpus == 0:
+            num_gpus = get_gpu_count_mxnet()
+            if num_gpus == 0:
+                num_gpus = get_gpu_count_torch()
+        return num_gpus
```

Comment:
Duplicated code - can we move this into some utility class?

Reply:
good point, updated